Flying to the Moon

katsuragiyy opened the pull request:
UAE, Green Ukraine, Samoa, New Zealand, Laos, Dahomey, Madagascar, Lao, Sarawak, Niue, Nauru, Kenya, Greenland, Brunei, and my sanity. all in one pull request!

katsuragiyy committed:
Github

katsuragiyy committed:
Merge branch 'Beta-Edits' into katsura-fly-me-to-the-moon

katsuragiyy committed:
Merge branch 'Beta-Edits' into katsura-fly-me-to-the-moon

katsuragiyy committed:
Cyprus

katsuragiyy committed:
im done for today

katsuragiyy committed:
stuff

katsuragiyy committed:
Skibiding my Toilet

Fixes to NZL and SAM, a few UAE & TAN stuff, and, obviously, Western Armenia and Southern Macedonia

katsuragiyy committed:
Redlight Redlight

katsuragiyy committed:
Gooning next to you!

katsuragiyy committed:
Samoa focus tree

April 7 TM

Commit:
s

Commit:
Revert "s"

This reverts commit e34e05bf965fbc696fb2fd793ce09b9da7af1315.

katsuragiyy committed:
Sigmaing next to you

Your wish shall be granted.

katsuragiyy committed:
Brunei Town

marijn211 commented:
[16:55:53][no_game_date][triggerimplementation.cpp:3427]: Invalid focus scripted in trigger. has_completed_focus = TAN_invite_japanese (common/national_focus/tuva.txt:1917: has_completed_focus)
[16:55:53][no_game_date][trigger.cpp:117]: Trigger failed to validate: common/national_focus/tuva.txt:1917: has_completed_focus

marijn211 committed:
Merge branch 'Beta-Edits' into katsura-fly-me-to-the-moon

marijn211 reviewed (changes requested):
Will check post 7th cuz new tree
(also please fix error above)

marijn211 reviewed (changes requested):
@Speculoos30 should comment on the Greeks and Turks why he commented them out

katsuragiyy committed:
Marshmello Islands

katsuragiyy committed:
s

katsuragiyy committed:
Final one i promise

katsuragiyy committed:
okay

katsuragiyy committed:
tap water

Brunei Madagascar and Benin (might take a break from dis after i finish all i have in mind to focus on my own projects)

katsuragiyy committed:
oh ya forgor this

pfp change no appear in alphabot 😔

katsuragiyy committed:
Merge branch 'Beta-Edits' into katsura-fly-me-to-the-moon

katsuragiyy committed:
arm arm arm

katsuragiyy committed:
branai

katsuragiyy committed:
carpathia

katsuragiyy committed:
fqfe

katsuragiyy committed:
stuff

katsuragiyy committed:
Hello Tanker. welcome to Github.

katsuragiyy committed:
12 changed files

qq

katsuragiyy committed:
ken

katsuragiyy committed:
edible water?

katsuragiyy committed:
qfujque

katsuragiyy committed:
typo

katsuragiyy committed:
Znachit - snova dozhdʹ i znakomyy veter

I opyatʹ odin on na tselom svete.

marijn211 reviewed (changes requested):
Error log

katsuragiyy committed:
fix fix fix

katsuragiyy committed:
thai-po

katsuragiyy committed:
rizz

katsuragiyy committed:
rifles

katsuragiyy committed:
kampuchea

katsuragiyy committed:
op

katsuragiyy committed:
its good now

katsuragiyy committed:
Merge branch 'Beta-Edits' into katsura-fly-me-to-the-moon

katsuragiyy committed:
Ooh-wee-hoo, I look just like Skibidi Toilet

oh-oh and your the Rizzler Sigma Alpha

katsuragiyy committed:
BAM-BAM knock on the toilet

katsuragiyy committed:
new portraits

marijn211 reviewed (dismissed):
[20:44:27][no_game_date][texturehandler.cpp:216]: Couldn't find texture file: 'gfx/leaders/KAL/portrait_KAL_nikolai_tundutov'
[20:44:29][no_game_date][texturehandler.cpp:160]: Texture Handler encountered missing texture file: gfx/interface/technologies/USA_basic_medium_tank_destroyer.dds
[20:44:29][no_game_date][texturehandler.cpp:216]: Couldn't find texture file: 'gfx/interface/technologies/USA_basic_medium_tank_destroyer.dds'
[20:44:29][no_game_date][texturehandler.cpp:160]: Texture Handler encountered missing texture file: gfx/interface/technologies/USA_improved_medium_tank_destroyer.dds
[20:44:29][no_game_date][texturehandler.cpp:216]: Couldn't find texture file: 'gfx/interface/technologies/USA_improved_medium_tank_destroyer.dds'
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 21" in file: "history/countries/BAS - British Antilles.txt" near line: 24
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/BAS - British Antilles.txt: 25
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/BAS - British Antilles.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 29" in file: "history/countries/BHR - Bahrain.txt" near line: 32
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/BHR - Bahrain.txt: 33
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/BHR - Bahrain.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 29" in file: "history/countries/BRB - Barbados.txt" near line: 32
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/BRB - Barbados.txt: 33
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/BRB - Barbados.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 26" in file: "history/countries/CHT - Chatham Islands.txt" near line: 29
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/CHT - Chatham Islands.txt: 30
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/CHT - Chatham Islands.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 16" in file: "history/countries/COM - Comoros.txt" near line: 19
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/COM - Comoros.txt: 20
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/COM - Comoros.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 21" in file: "history/countries/CRC - Curacao.txt" near line: 24
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/CRC - Curacao.txt: 25
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/CRC - Curacao.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 25" in file: "history/countries/DMN - Dominica.txt" near line: 28
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/DMN - Dominica.txt: 29
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/DMN - Dominica.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 27" in file: "history/countries/DOM - Dominican Republic.txt" near line: 30
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/DOM - Dominican Republic.txt: 31
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/DOM - Dominican Republic.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 30" in file: "history/countries/EIS - Lesser Sunda.txt" near line: 33
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/EIS - Lesser Sunda.txt: 34
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/EIS - Lesser Sunda.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 20" in file: "history/countries/FAR - Faroes.txt" near line: 23
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/FAR - Faroes.txt: 24
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/FAR - Faroes.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 34" in file: "history/countries/FIJ - Fiji.txt" near line: 37
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/FIJ - Fiji.txt: 38
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/FIJ - Fiji.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 21" in file: "history/countries/FSM - Micronesia.txt" near line: 24
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/FSM - Micronesia.txt: 25
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/FSM - Micronesia.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 21" in file: "history/countries/GDL - Guadeloupe.txt" near line: 24
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/GDL - Guadeloupe.txt: 25
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/GDL - Guadeloupe.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 29" in file: "history/countries/GRA - Grenada.txt" near line: 32
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/GRA - Grenada.txt: 33
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/GRA - Grenada.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 30" in file: "history/countries/GRL - Greenland.txt" near line: 33
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/GRL - Greenland.txt: 34
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/GRL - Greenland.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 24" in file: "history/countries/GUM - Guam.txt" near line: 27
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/GUM - Guam.txt: 28
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/GUM - Guam.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 22" in file: "history/countries/HAI - Haiti.txt" near line: 25
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/HAI - Haiti.txt: 26
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/HAI - Haiti.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 49" in file: "history/countries/HAW - Hawaii.txt" near line: 52
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/HAW - Hawaii.txt: 53
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/HAW - Hawaii.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 21" in file: "history/countries/KIR - Kiribati.txt" near line: 24
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/KIR - Kiribati.txt: 25
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/KIR - Kiribati.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 29" in file: "history/countries/MAA - Sint Maarten.txt" near line: 32
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/MAA - Sint Maarten.txt: 33
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/MAA - Sint Maarten.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 33" in file: "history/countries/MAD - Madagascar.txt" near line: 36
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/MAD - Madagascar.txt: 37
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/MAD - Madagascar.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 25" in file: "history/countries/MAR - Martinique.txt" near line: 28
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/MAR - Martinique.txt: 29
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/MAR - Martinique.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 16" in file: "history/countries/MAU - Mauritius.txt" near line: 19
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/MAU - Mauritius.txt: 20
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/MAU - Mauritius.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 25" in file: "history/countries/MLD - Maldives.txt" near line: 28
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/MLD - Maldives.txt: 29
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/MLD - Maldives.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 21" in file: "history/countries/MLK - Maluku.txt" near line: 24
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/MLK - Maluku.txt: 25
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/MLK - Maluku.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 20" in file: "history/countries/MLT - Malta.txt" near line: 23
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/MLT - Malta.txt: 24
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/MLT - Malta.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 26" in file: "history/countries/MRS - Marshall Islands.txt" near line: 29
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/MRS - Marshall Islands.txt: 30
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/MRS - Marshall Islands.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 27" in file: "history/countries/NIU - Niue.txt" near line: 30
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/NIU - Niue.txt: 31
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/NIU - Niue.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 22" in file: "history/countries/NRU - Nauru.txt" near line: 25
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/NRU - Nauru.txt: 26
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/NRU - Nauru.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 21" in file: "history/countries/PLU - Palau.txt" near line: 24
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/PLU - Palau.txt: 25
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/PLU - Palau.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 20" in file: "history/countries/PNG - Papua New Guinea.txt" near line: 23
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/PNG - Papua New Guinea.txt: 24
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/PNG - Papua New Guinea.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 37" in file: "history/countries/PPA - Papua.txt" near line: 40
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/PPA - Papua.txt: 41
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/PPA - Papua.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 53" in file: "history/countries/PUE - Puerto Rico.txt" near line: 56
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/PUE - Puerto Rico.txt: 57
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/PUE - Puerto Rico.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 16" in file: "history/countries/SAO - Sao Tome.txt" near line: 19
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/SAO - Sao Tome.txt: 20
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/SAO - Sao Tome.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 21" in file: "history/countries/SEY - Seychelles.txt" near line: 24
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/SEY - Seychelles.txt: 25
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/SEY - Seychelles.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 20" in file: "history/countries/SOL - Solomon Islands.txt" near line: 23
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/SOL - Solomon Islands.txt: 24
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/SOL - Solomon Islands.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 29" in file: "history/countries/STK - Saint Kitts & Nevis.txt" near line: 32
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/STK - Saint Kitts & Nevis.txt: 33
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/STK - Saint Kitts & Nevis.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 29" in file: "history/countries/STL - Saint Lucia.txt" near line: 32
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/STL - Saint Lucia.txt: 33
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/STL - Saint Lucia.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 29" in file: "history/countries/SVG - Saint Vincent & the Grenadines.txt" near line: 32
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/SVG - Saint Vincent & the Grenadines.txt: 33
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/SVG - Saint Vincent & the Grenadines.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 22" in file: "history/countries/TAH - Tahiti.txt" near line: 25
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/TAH - Tahiti.txt: 26
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/TAH - Tahiti.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 25" in file: "history/countries/TML - Timor Leste.txt" near line: 28
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/TML - Timor Leste.txt: 29
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/TML - Timor Leste.txt
[20:44:29][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 20" in file: "history/countries/TRI - Trinidad and Tobago.txt" near line: 23
[20:44:29][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/TRI - Trinidad and Tobago.txt: 24
[20:44:29][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/TRI - Trinidad and Tobago.txt
[20:44:30][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 25" in file: "history/countries/TUV - Tuvalu.txt" near line: 28
[20:44:30][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/TUV - Tuvalu.txt: 29
[20:44:30][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/TUV - Tuvalu.txt
[20:44:30][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 21" in file: "history/countries/VAN - Vanuatu.txt" near line: 24
[20:44:30][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/VAN - Vanuatu.txt: 25
[20:44:30][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/VAN - Vanuatu.txt
[20:44:30][no_game_date][effect.cpp:352]: Error: "Unexpected limit in an else block: limit, near line: 18" in file: "history/countries/ZAN - Zanzibar.txt" near line: 21
[20:44:30][no_game_date][effect.cpp:386]: Else/else if not following an if or else if in history/countries/ZAN - Zanzibar.txt: 22
[20:44:30][no_game_date][countryhistory.cpp:170]: Unknown History Command ==>'mtg_transport'<==  for Countryhistory/countries/ZAN - Zanzibar.txt

katsuragiyy committed:
Never mind!

katsuragiyy commented:
@marijn2112 these errors are not related to my branch.
[20:44:29][no_game_date][texturehandler.cpp:160]: Texture Handler encountered missing texture file: gfx/interface/technologies/USA_basic_medium_tank_destroyer.dds
[20:44:29][no_game_date][texturehandler.cpp:216]: Couldn't find texture file: 'gfx/interface/technologies/USA_basic_medium_tank_destroyer.dds'
[20:44:29][no_game_date][texturehandler.cpp:160]: Texture Handler encountered missing texture file: gfx/interface/technologies/USA_improved_medium_tank_destroyer.dds
[20:44:29][no_game_date][texturehandler.cpp:216]: Couldn't find texture file: 'gfx/interface/technologies/USA_improved_medium_tank_destroyer.dds'

I have not touched GFX. 
anyhow, i decided to remove the things causing the other errors errors (Even tho i could have probably fixed them, but i really can't bother)

Simpyslav committed:
Merge branch 'Beta-Edits' into katsura-fly-me-to-the-moon

katsuragiyy committed:
saint vincent and the grenadines

katsuragiyy committed:
Merge branch 'katsura-fly-me-to-the-moon' of https://github.com/deliciousmods/1956_beta into katsura-fly-me-to-the-moon

katsuragiyy committed:
sam

katsuragiyy committed:
Update strategic_region_names_l_english.yml

katsuragiyy committed:
Thai-iti

I've got to stop with the Thai jokes

katsuragiyy committed:
Merge branch 'Beta-Edits' into katsura-fly-me-to-the-moon

Simpyslav committed:
New Zealand Focus Tree Touchups

Reworded Focus Tree Descriptions and  Ideas

Reworked Army/Air/Navy Tree
Expanded Industrial Tree

Added flavour.

Simpyslav committed:
Merge branch 'katsura-fly-me-to-the-moon' of https://github.com/deliciousmods/1956_beta into katsura-fly-me-to-the-moon

katsuragiyy committed:
South Africa focus tree rework

katsuragiyy committed:
Q

Simpyslav committed:
2/3 Pushes

New Zealand General Focus Tree "Upgrade"

Finalized Army/Air/Eco Tree

katsuragiyy committed:
samoa

katsuragiyy committed:
VP change

in the base game its worth 10

Simpyslav committed:
3/3

Will update, upon review and concerns.
Any suggestion welcome. <3

katsuragiyy committed:
HUN portrait

katsuragiyy committed:
Merge branch 'Beta-Edits' into katsura-fly-me-to-the-moon

katsuragiyy committed:
TUR navy amigos

katsuragiyy committed:
TUR P2

which bedroom are you sleeping in?
1.
2.
3.

Commit:
Fix

TheBrit7 committed:
Update TUR.txt

Simpyslav committed:
Shh nothing to see here

Updated based on feedback, Increased the Industrial Tree.

Simpyslav committed:
Shh nothing to see here

DutchPlanDerLindeSus committed:
stuff

katsuragiyy committed:
alright folks i got school

katsuragiyy committed:
Update BLR.txt

katsuragiyy committed:
h

Simpyslav committed:
Merge branch 'Beta-Edits' into katsura-fly-me-to-the-moon

Simpyslav committed:
Shhh

katsuragiyy committed:
plz merge

katsuragiyy committed:
Merge branch 'Beta-Edits' into katsura-fly-me-to-the-moon

marijn211 reviewed (commented):
Please dont reopen while error log is not clean

Simpyslav committed:
Clearing up Errorlog

Missing BLR fix and Kenya

Simpyslav committed:
Merge branch 'Beta-Edits' into katsura-fly-me-to-the-moon

Simpyslav committed:
Loc + Checks

Simpyslav committed:
Merge branch 'Beta-Edits' into katsura-fly-me-to-the-moon

katsuragiyy committed:
update

marijn211 committed:
Fix missing capital letters

Harain committed:
add small versions of kr gfx

a cost was also duplicated in BLR.txt
and also a missing kr gfx in GRE.txt

katsuragiyy committed:
rada stuff

marijn211 committed:
Branch hiding for NZL
